### PR TITLE
Add a try out component for SSE and WS APIs.

### PR DESCRIPTION
**Pull request by sachithKay**
- Adds a tryout view for SSE APIs.
- Adds a tryout view for WS APIs.
Screen Shot 2021-03-16 at 10 43 26 PM
Screen Shot 2021-03-16 at 10 42 56 PM

### CODE REVIEW

File: features/apimgt/org.wso2.carbon.apimgt.store.feature/src/main/resources/devportal/source/src/app/components/Apis/Details/AsyncApiConsole/GenericSubscriptionUI.jsx [hunk 1]
```diff
@@ -0,0 +1,118 @@
...
+            padding: 0,
+            'label + &': {
+                marginTop: theme.spacing(1),
+            },
+        },
+        bootstrapCurl: {
+            borderRadius: 4,
+            backgroundColor: theme.custom.curlGenerator.backgroundColor,
+            color: theme.custom.curlGenerator.color,
+            border: '1px solid #ced4da',
```
Comment (tmkasun) [inline features/apimgt/org.wso2.carbon.apimgt.store.feature/src/main/resources/devportal/source/src/app/components/Apis/Details/AsyncApiConsole/GenericSubscriptionUI.jsx:46]: Better to uses MUI [<Box/>](https://material-ui.com/system/shadows/#shadows) for these

File: features/apimgt/org.wso2.carbon.apimgt.store.feature/src/main/resources/devportal/source/src/app/components/Apis/Details/AsyncApiConsole/GenericSubscriptionUI.jsx [hunk 2]
```diff
@@ -0,0 +1,118 @@
...
+                        }}
+                        InputLabelProps={{
+                            shrink: true,
+                            className: classes.bootstrapFormLabel,
+                        }}
+                    />
+                </Grid>
+            </AccordionDetails>
+            <AccordionActions style={{ paddingRight: '18px' }}>
+                <CopyToClipboard text={command} onCopy={() => Alert.info('cURL copied')}>
```
Comment (tmkasun) [inline features/apimgt/org.wso2.carbon.apimgt.store.feature/src/main/resources/devportal/source/src/app/components/Apis/Details/AsyncApiConsole/GenericSubscriptionUI.jsx:110]: Text should be internationalised with FormattedMessage

**Review by tmkasun — APPROVED** [review]

File: features/apimgt/org.wso2.carbon.apimgt.store.feature/src/main/resources/devportal/source/src/app/components/Apis/Details/AsyncApiConsole/GenericSubscriptionUI.jsx (hunk 1 above)
Comment (sachithKay) [reply features/apimgt/org.wso2.carbon.apimgt.store.feature/src/main/resources/devportal/source/src/app/components/Apis/Details/AsyncApiConsole/GenericSubscriptionUI.jsx:46]: This is the format followed for all TextFields in the portal. Shall we leave this for consistency?
https://github.com/wso2/carbon-apimgt/blob/8f90d1304b58789a65329438fe0c08d9c336065a/features/apimgt/org.wso2.carbon.apimgt.store.feature/src/main/resources/devportal/source/src/app/components/Apis/Details/Environments.jsx#L70

Comment (tmkasun) [reply features/apimgt/org.wso2.carbon.apimgt.store.feature/src/main/resources/devportal/source/src/app/components/Apis/Details/AsyncApiConsole/GenericSubscriptionUI.jsx:46]: Well, then this should come from the theme @chanaka3d  ?

**Pull request merged**